### PR TITLE
feat: notebook for custom evaluations

### PR DESCRIPTION
**Pull request by andrewferlitsch**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
New notebook for adding a custom evaluation to a model in the Model Registry
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/model_evaluation/get_started_with_custom_model_evaluation_import.ipynb [hunk 1]
```diff
@@ -0,0 +1,708 @@
...
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "NIq7R4HZCfIc"
+      },
+      "outputs": [],
+      "source": [
+        "! gsutil mb -l $REGION -p $PROJECT_ID $BUCKET_URI"
```
Comment (yil532) [inline notebooks/official/model_evaluation/get_started_with_custom_model_evaluation_import.ipynb:366]: This line looks weird. Could we please change the syntax?
https://screenshot.googleplex.com/3SXwVtLwtxjRHf4

File: notebooks/official/model_evaluation/get_started_with_custom_model_evaluation_import.ipynb [hunk 2]
```diff
@@ -0,0 +1,710 @@
+{
```
Comment (telpirion) [inline notebooks/official/model_evaluation/get_started_with_custom_model_evaluation_import.ipynb:1]: <p>nit: you might also provide a link to a calculator to estimate costs of accelerator usage.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (telpirion) [inline notebooks/official/model_evaluation/get_started_with_custom_model_evaluation_import.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    TF = "2.5".replace(".", "-")
</pre><p>question: Won't <code>TF[0]</code> always evaluate to "2"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (telpirion) [inline notebooks/official/model_evaluation/get_started_with_custom_model_evaluation_import.ipynb:1]: <p>typo: "distinquihed" -&gt; 'distinguished'. I would also remove the semi-colon.</p><p> </p><p>suggestion:</p><p>"The Vertex AI Model Registry supports importing multiple model evaluations for a model, where each evaluation is distinguished by a unique display name."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

**Review by telpirion — APPROVED** [review]

File: notebooks/official/model_evaluation/get_started_with_custom_model_evaluation_import.ipynb (hunk 1 above)
Comment (andrewferlitsch) [reply notebooks/official/model_evaluation/get_started_with_custom_model_evaluation_import.ipynb:368]: rendering issue in git

File: notebooks/official/model_evaluation/get_started_with_custom_model_evaluation_import.ipynb (hunk 2 above)
Comment (andrewferlitsch) [reply notebooks/official/model_evaluation/get_started_with_custom_model_evaluation_import.ipynb:1]: added

Comment (andrewferlitsch) [reply notebooks/official/model_evaluation/get_started_with_custom_model_evaluation_import.ipynb:1]: updated

**Pull request merged**